EventFilter/SiPixelRawToDigi/plugins/SiPixelRawToDigi.cc

```diff
@@ -97,6 +98,9 @@ SiPixelRawToDigi::SiPixelRawToDigi( const edm::ParameterSet& conf )
     usePhase1 = config_.getParameter<bool> ("UsePhase1");
     if(usePhase1) edm::LogInfo("SiPixelRawToDigi")  << " Use pilot blade data (FED 40)";
   }
+  //CablingMap could have a label //Tav
+  CablingMapLabel = ""; 
+  CablingMapLabel = config_.getParameter<std::string> ("CablingMapLabel");
```

Comment:
@tvami: Since you deleted the `if` statement, you should also delete line 102, which is no longer needed. Line 103 assigns to `CablingMapLabel` and makes line 102 pointless.